templates/multipass/install.html

```diff
@@ -0,0 +1,329 @@
+      <hr class="p-rule" />
+      <div class="p-section--shallow">
+        <h2>Select your OS to get started</h2>
+      </div>
+      <div class="js-tabbed-content">
+        <div class="p-section--shallow">
+          <div class="p-tabs">
+            <div class="p-tabs__list"
+                 role="tablist"
+                 aria-label="Multipass installation instructions">
+              <div class="p-tabs__item">
+                <button class="p-tabs__link"
+                        role="tab"
+                        aria-selected="true"
+                        aria-controls="linux-tab"
+                        id="linux"
+                        data-maintain-hash="linux">Linux</button>
+              </div>
+              <div class="p-tabs__item">
+                <button class="p-tabs__link"
+                        role="tab"
+                        aria-selected="false"
+                        aria-controls="windows-tab"
+                        id="windows"
+                        data-maintain-hash="windows"
+                        tabindex="-1">Windows</button>
+              </div>
+              <div class="p-tabs__item">
+                <button class="p-tabs__link"
+                        role="tab"
+                        aria-selected="false"
+                        aria-controls="mac-tab"
+                        id="mac"
+                        data-maintain-hash="mac"
+                        tabindex="-1">MacOS</button>
+              </div>
+            </div>
+          </div>
+          <!-- tab content -->
+          <div id="linux-tab" role="tabpanel" aria-labelledby="linux" tabindex="0">
+            <div class="row--50-50">
+              <div class="col">
+                <h3>Install Multipass on Linux</h3>
+              </div>
+              <div class="col">
+                <p>To install Multipass on a Linux system, run the following command:</p>
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block--icon"><code>sudo snap install multipass</code></pre>
+                </div>
+                <p>
+                  Don't have the <code>snap</code> command? <a href="https://docs.snapcraft.io/installing-snapd">Get set up for snaps</a>.
+                </p>
+              </div>
+            </div>
+          </div>
+
+          <div id="windows-tab"
+               role="tabpanel"
+               aria-labelledby="windows"
+               tabindex="0">
+            <div class="row--50-50">
+              <div class="col">
+                <h3>Install Multipass on Windows</h3>
+              </div>
+              <div class="col">
+                <ol class="p-stepped-list u-no-margin--left">
+                  <li class="p-stepped-list__item u-no-padding--bottom">
+                    <p class="p-stepped-list__title p-heading--5 u-no-margin--bottom">
+                      <a href="https://multipass.run/download/windows">Download Multipass for Windows</a>
```

Comment:
I think you mentioned this in standup, but if we decommission the site wont these fail?

Reply:
Yup, I opened a ticket [here](https://warthogs.atlassian.net/browse/WD-17382)